pasqal_cloud/__init__.py

```diff
@@ -359,6 +367,36 @@ def cancel_batch(self, id: str) -> Batch:
             raise BatchCancellingError(e) from e
         return Batch(**batch_rsp, _client=self._client)
 
+    def cancel_batches(self, batch_ids: List[str]) -> BatchCancellationResponse:
+        """
+        Cancel a group of batches by their ids.
+
+        Args:
+            batch_ids: batch ids to cancel.
+
+        Returns:
+            BatchCancellationResponse:
+            a class containing the batches that have been cancelled and the id of
+            the batches that could not be cancelled with the reason explained
+
+        Raises:
+            BatchCancellingError which spawns from a HTTPError
+
+        """
+
+        try:
+            response = self._client.cancel_batches(
+                batch_ids=batch_ids,
+            )
+        except HTTPError as e:
+            raise BatchCancellingError(e) from e
```

Comment:
```suggestion
            raise BatchCancellingError("a user-friendly error message") from e
```
This provides a user-friendly message while retaining the original exception context for debugging.

Using `from e` links the original exception (e) as the cause of `BatchCancellingError`, so including e.message directly in BatchCancellingError isn’t necessary.